config/sentry.php

```diff
@@ -58,7 +58,7 @@
         // Capture SQL queries as breadcrumbs
         'sql_queries' => env('SENTRY_BREADCRUMBS_SQL_QUERIES_ENABLED', true),
 
-        // Capture SQL query bindings (parameters) in SQL query breadcrumbs
+        // Capture bindings in SQL query breadcrumbs. `true` or `append` shows them separate, `embed` inlines them
```

Comment:
I shortened the prior part of the description to fit everything on a single short line still.